solr/webapp/web/WEB-INF/web.xml

```diff
@@ -123,10 +129,16 @@
     </init-param>
   </servlet>
 
+  <!-- Fullstory additions -->
   <servlet-mapping>
     <servlet-name>Statz</servlet-name>
     <url-pattern>/admin/statz</url-pattern>
   </servlet-mapping>
+  <servlet-mapping>
```

Comment:
super minor spacing nit - do we want spaces before and after the new entry for consistency?

Reply:
idc, I just wanted to box off our own stuff from all the stock solr stuff, and omitting newlines made it visually more distinct for me